app/src/main/java/com/livelike/engagementsdksample/customwidgets/poll/CustomPollWidget.kt

```diff
@@ -0,0 +1,207 @@
+                    context,
+                    R.drawable.custom_progress_color_options_selected
+                )
+                holder.itemView.textView2.setTextColor(Color.WHITE)
+                holder.itemView.textView.setTextColor(Color.WHITE)
+            } else {
+                holder.itemView.lay_poll_img_option.setBackgroundResource(R.drawable.option_item_background)
+                holder.itemView.progressBar.progressDrawable = ContextCompat.getDrawable(
+                    context,
+                    R.drawable.custom_progress_color_options_selected
+                )
+                holder.itemView.textView2.setTextColor(Color.BLACK)
+                holder.itemView.textView.setTextColor(Color.BLACK)
+            }
+            holder.itemView.lay_poll_img_option.setOnClickListener {
+                selectedIndex = holder.adapterPosition
+                pollListener?.onSelectOption(item.id!!)
+                notifyDataSetChanged()
+            }
+        } else {
+            if (optionIdCount.containsKey(item.id)) {
+                holder.itemView.txt_percent.visibility = View.VISIBLE
+                holder.itemView.progressBar_text.visibility = View.VISIBLE
+                val total = optionIdCount.values.reduce { acc, i -> acc + i }
+                val percent = (optionIdCount[item.id!!]!!.toFloat() / total.toFloat()) * 100
+                holder.itemView.txt_percent.text = "$percent %"
+                holder.itemView.progressBar_text.progress = percent.toInt()
+            } else {
+                holder.itemView.txt_percent.visibility = View.INVISIBLE
+                holder.itemView.progressBar_text.visibility = View.INVISIBLE
+            }
+            holder.itemView.text_poll_item.text = "${item.description}"
+            if (selectedIndex == index) {
+                holder.itemView.lay_poll_text_option.setBackgroundResource(R.drawable.option_item_background_selected)
+                holder.itemView.text_poll_item.setTextColor(Color.WHITE)
+                holder.itemView.txt_percent.setTextColor(Color.WHITE)
+                holder.itemView.progressBar_text.progressDrawable = ContextCompat.getDrawable(
+                    context,
+                    R.drawable.custom_progress_color_options_selected
+                )
+            } else {
+                holder.itemView.lay_poll_text_option.setBackgroundResource(R.drawable.option_item_background)
+                holder.itemView.text_poll_item.setTextColor(Color.BLACK)
+                holder.itemView.txt_percent.setTextColor(Color.BLACK)
+                holder.itemView.progressBar_text.progressDrawable = ContextCompat.getDrawable(
+                    context,
+                    R.drawable.custom_progress_color_options
+                )
+            }
+            holder.itemView.lay_poll_text_option.setOnClickListener {
+                if (!isFollowUp) {
```

Comment:
what is this isFollowUp here? Remove it, Client will get confused when taking help from this code.